app/views/layouts/_menu.html.erb

```diff
@@ -6,62 +6,61 @@
     </li>
     <li class="dropdown active">
       <a href="#" class="dropdown-toggle" data-toggle="dropdown" data-hover="dropdown" data-delay="0" data-close-others="false">
-        Sektionen <%= fa_icon('angle-down') %>
+      <%= t('menu.guild') %> <%= fa_icon('angle-down') %>
     </a>
     <ul class="dropdown-menu">
       <% if (sekt_menus = menus_by_location['Sektionen']).present? %>
         <%= render partial: 'menus/menu', collection: sekt_menus %>
       <% else %>
-        <li><%= link_to 'Om oss', :om %></li>
-        <li><%= link_to 'Utskott', :councils %></li>
-        <li><%= link_to 'Dokument', documents_path %></li>
+        <li><%= link_to(t('menu.about'), :om) %></li>
+        <li><%= link_to(model_name(Council), :councils) %></li>
+        <li><%= link_to(model_name(Document), :documents) %></li>
```

Comment:
This is not 100% DRY. I've used this helper in my past projects to avoid this kind of repetition:

```
 def link obj, options = {}
    if obj.nil?
      content_tag :i, '[missing]'
    elsif obj.is_a?(ActiveRecord::Base) && obj.respond_to?(:name)
      link_to obj.name, obj, options
    elsif obj.respond_to? :model_name
      link_to obj.model_name.human(:count => 2), obj
    else
      link_to obj, obj, options
    end
  end
```

Maybe you should consider importing that (or similar). The "name" interface is really useful and should be part of the framework IMO.